lib/pure/json.nim

```diff
@@ -761,7 +761,9 @@ proc `==`* (a,b: JsonNode): bool =
     of JObject:
       a.fields == b.fields
 
-proc hash* (n:JsonNode): Hash =
+proc hash*(n: Table[string, JsonNode]): Hash
```

Comment:
Annotate it with {.noSideEffect.} or at least .gcsafe, locks: 0   I know, I know, annoying.